### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mdarnall**
Added some additional examples using proxies and iTunes Transporter. 

This hopefully will save folks time as it took me a while how to configure a proxy on our Jenkins environment that is behind a firewall, without having access to update slave installs of the uploader application.

### CODE REVIEW

**Comment by KrauseFx** [comment]
Where do you have that information from? Maybe we could link to to the documentation instead?

**Comment by KrauseFx** [comment]
Thanks for the pull request, this looks great otherwise 👍

**Comment by mdarnall** [comment]
It was after I looked at the already linked Java document I realised it could be specified on the command-line. 

```

File: deliver/README.md [hunk 1]
```diff
@@ -250,6 +250,11 @@ TOOLS_PATH=$( xcode-select -p )
 REL_PATH='../Applications/Application Loader.app/Contents/itms/java/lib/net.properties'
 echo "$TOOLS_PATH/$REL_PATH"
 ```
+As an alternative to editing the properties files, proxy configuration can be specified on the command line directly:
```
Comment (KrauseFx) [inline deliver/README.md]: Is the order correct? Should this be below the proxy configuration link below?

Comment (mdarnall) [inline deliver/README.md]: Yeah, good call. Updated.

**Review by KrauseFx — APPROVED** [review]

**Comment by KrauseFx** [comment]
Sorry CI is failing, could you rebase from master or submit a new PR with these changes? Thanks

**Comment by mdarnall** [comment]
rebased..

**Comment by KrauseFx** [comment]
Beautiful, thanks 👍

**Pull request merged**